lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
```

Comment:
maybe instead of defaulting to 400 this should come come from response_values[:status] if available, but i'm weary of blindly passing status returned from downstream. For example a 500 error from downstream is technically a 502 since we're acting as a proxy to that response.